### PR TITLE
Page for Telegram client integration

### PR DESCRIPTION
**Pull request by lufton**
## Proposed change
Page for new Telegram client integration

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/121951

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Added integration for Telegram client using the Telethon library, allowing message management, event capturing, and user/bot configuration.
  - Introduced multiple service data attributes, new events, sensors, and binary sensors for Telegram client integration.
  - Provided automation examples for various scenarios involving Telegram messaging and interactions.

- **Documentation**
  - Updated documentation to include details on new integrations and features for several platforms, including Alexa, Aprilaire, August, Autarco, Bang & Olufsen, Broadlink, Command Line, DoorBird, Google Sheets, Group, Habitica, History Stats, HomeKit, Hydrawise, Israel Rail, madVR, Matter, Mealie, MINI Connected, Netatmo, Ollama, Roborock, Russound RIO, Shelly, SimpleFIN, Switch Template, Technove, Telegram Client, Template, Teslemetry, Tessie, and UniFi Protect.

### CODE REVIEW

File: source/_integrations/telegram_client.markdown [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+## Options flow
+
+Telegram client integration supports options on events that are going to propagate to Home Assistant event bus via **Configure** button on [this page](https://my.home-assistant.io/redirect/integration/?domain=telegram_client). You can select witch kinds of [events](#events) you want to be able to listen for.
+Each event has its own set of settings. Please refer to [Telethon events documentation page](https://docs.telethon.dev/en/stable/modules/events.html) for full explanation.
+
+## Services
+
+* [telegram_client.send_messages](#service-telegram_clientsend_messages) – Send message(s)
+* [telegram_client.edit_message](#service-telegram-clientedit-message) – Edit message
+* [telegram_client.delete_messages](#service-telegram-clientdelete-messages) – Delete message(s)
```
[the remark this answers is not in the document]
Comment (lufton) [reply source/_integrations/telegram_client.markdown]: Fixed in latest commit

File: source/_integrations/telegram_client.markdown [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+      config_entry_id: "{{ trigger.event.data.config_entry_id }}"
+      message_ids: "{{ trigger.event.data.max_id }}"
+mode: restart
+
+```
+
+### Respond to a callback query
+
+
+
```
[the remark this answers is not in the document]
Comment (lufton) [reply source/_integrations/telegram_client.markdown]: Resolved in latest commit

[the remark this answers is not in the document]
Comment (lufton) [reply source/_integrations/telegram_client.markdown:239]: Resolved in latest commit

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**